keps/sig-release/1731-publishing-packages/README.md

```diff
@@ -63,64 +62,79 @@ This all prolongs the release process, it is a hard dependency on a small group
 
 ### Goals
 
-The whole process should be folded into the release tooling, it should be part of the release process, and should not involve anyone other than the release branch / patch release team.
+The whole process should be folded into the release tooling, it should be part of the release process, and should not involve anyone other than the release managers.
 For each release the release team cuts, packages should also be generated and published automatically.
 
 There should be multiple channels users can subscribe to: **stable**, **dev**, and **nightly**.
 
-
 ### Non-Goals
 
-The actual package generation is a different problem that is discussed in [this KEP][pkg-gen-kep].
+The original version of this KEP had the following statement:
+
+> The actual package generation is a different problem that is discussed in [this KEP][pkg-gen-kep].
+
+Based on recent review of this KEP @saschagrunert wants SIG Release to own
+the process of automatically building and publishing the packages and this will
+implicitly aceive the main goal of [#2503](https://github.com/kubernetes/enhancements/issues/2503).
 
 [pkg-gen-kep]: https://github.com/kubernetes/enhancements/pull/858
 
 ## Proposal
 
 - Make the infrastructure generic and simple enough to be easily handed off to the CNCF
-    - Storage buckets (to store the staged/released packages)
-    - DNS entries (e.g. apt.kubernetes.io, ...)
-    - package mirror (e.g. a self hosted aptly/artifactory/... or as a service)
-        - have multiple channels, e.g. `stable`, `dev`, `nightly`
-- Run the [package building][pkg-gen-kep] as part of the staging process
-    - on GCB / not on an individual's machine
+  - Storage buckets (to store the staged/released packages)
+  - DNS entries (e.g. apt.kubernetes.io, ...)
+  - package mirror (e.g. a self hosted aptly/artifactory/... or as a service)
+    - have multiple channels, e.g. `stable`, `dev`, `nightly`
+- Run the package builds as part of [krel stage and release]
 - Have a safe way to store the signing key and make it available to the release team and release tooling
```

Comment:
Can we log a concern around this (maybe after merge)? It's not really fleshed out enough.

Reply:
Yeah I think it really depends on the technical solution we choose.